src/hotspot/share/jfr/jni/jfrJniMethod.cpp

```diff
@@ -401,3 +401,7 @@ JVM_ENTRY_NO_ENV(jlong, jfr_host_total_memory(JNIEnv* env, jobject jvm))
   return os::physical_memory();
 #endif
 JVM_END
+
+JVM_ENTRY_NO_ENV(void, jfr_emit_data_loss(JNIEnv* env, jobject jvm, jlong bytes))
+  EventDataLoss::commit(bytes, 0L);
```

Comment:
If a value is missing, we use min_jlong to indicate not available or not applicable